### DISCUSSION TITLE
Example using react-three-fiber's instancedMesh

### DISCUSSION THREAD

**Discussion by jasonlong** [pmndrs/react-spring#1539] [Support]
Would anyone have a simple example of using react-spring with an `instancedMesh` in r3f? I'm trying to create a demo where several cubes have the same spring config and activate on hover. Thank you in advance! 🙏🏼

**Comment by creativeSpecies** [comment]
Hello @jasonlong, actually I think I'm trying to do the same thing as you :
[https://codesandbox.io/s/instanced-mesh-grid-react-spring-pvsw5
](url)

This part is responsible for the hovered instance, but I don't see how to trigger it

```js
const scale = id === hovered ? 3 : 1
CONFIG.tempObject.position.set(x, 0, y)
CONFIG.tempObject.scale.y = scale
CONFIG.tempObject.updateMatrix()
meshRef.current.setMatrixAt(id, CONFIG.tempObject.matrix)

**Comment by drcmda** [comment]
you'll find examples here: https://docs.pmnd.rs/react-three-fiber/examples/basics
and here: https://docs.pmnd.rs/react-three-fiber/examples/showcase

**Comment by creativeSpecies** [comment]
Thanks for the examples, I already looked at them.

I found a brute workaround, but it nearly works... now, the only thing that remains, is to animate reverse the scale, once out :
[https://codesandbox.io/s/instanced-mesh-grid-react-spring-onchange-bfhtm](https://codesandbox.io/s/instanced-mesh-grid-react-spring-onchange-bfhtm)
